web/src/main/resources/application.yml

```
@@ -12,11 +12,11 @@ spring:
 
   mail:
     # Credentials for the gmail smtp service passed in run configurations
-    site-email:
-    host:
-    port:
-    username:
-    password:
+    site-email: wilmaproject.dev@gmail.com
+    host: smtp.gmail.com
+    port: 587
+    username: wilmaproject.dev@gmail.com
+    password: qxnxokfdfwhqbnfd
```

Personally I don't think this is a good idea being that the repo is public
I mean it's already in the commits, but if it's not in plain sight then maybe that will be enough ?
```suggestion
    site-email:
    host:
    port:
    username:
    password:
```